### ISSUE TITLE
Failure: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by JannikZed** [nathancahill/aws-serverless-micro#5]
Don't know, what I'm doing wrong, but when following exactly your example in the README I only receive the answer/error 
```
Serverless: Failure: Cannot read property 'length' of undefined
TypeError: Cannot read property 'length' of undefined
    at /Users/XXX/Documents/git/XXX/node_modules/aws-serverless-micro/index.js:117:24
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```
Do you have an idea, what might face this issue?

**Comment by ozgreat** [comment]
Have the same issue

**Comment by nahtnam** [comment]
Same here, getting this when running on Netlify (based on AWS lambda). I swear I had it working before though...

**Comment by nathancahill** [comment]
Was able to replicate. It looks like something changed in Node 11 and I'm updating the library to be compatible.

**Comment by nathancahill** [comment]
Lambda started supporting Node 12 so that's when the failure was introduced: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

**Comment by nathancahill** [comment]
This has been resolved in v1.0.3.

**Comment by nathancahill** [comment]
v1.0.4 rather.
